php/admin/class-license.php

```diff
@@ -155,12 +167,29 @@ public function license_success_message() {
 		return sprintf( '<div class="notice notice-success"><p>%s</p></div>', esc_html( $message ) );
 	}
 
+	/**
+	 * Admin notice for the license request failing.
+	 *
+	 * This is for when the validation request fails entirely, like with a 404.
+	 * Not for when it returns that the license is invalid.
+	 *
+	 * @return string
+	 */
+	public function license_request_failed_message() {
+		$message = sprintf(
+			/* translators: %s is the email link for support */
+			__( 'There was a problem activating the license, but it may not be invalid. If the problem persists, please <a href="%s">contact support</a>.', 'block-lab' ),
+			'mailto:hi@getblocklab.com?subject=There was a problem activating my Block Lab Pro license'
```

Comment:
It makes sense to leave the subject untranslated. 👍

Reply:
Haha, yeah.